recipes/lmdbxx/meta.yaml

```
@@ -0,0 +1,35 @@
+{% set name = "lmdbxx" %}
+{% set version = "0.9.14.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/drycpp/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.bz2
+  sha256: 8121cc0a3f856584c5ce79638614468e2aba617d0338d14871f4ec7acbee2bec
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    
+test:
+  commands:
+    - test -e ${PREFIX}/include/lmdb++.h  # [unix]
+    - if not exist %LIBRARY_PREFIX%\include\lmdb++.h (exit 1)  # [win]
+
+about:
+  home: https://github.com/drycpp/lmdbxx  
+  summary: This is a comprehensive C++ wrapper for the LMDB embedded database library,
+  description: This is a comprehensive C++ wrapper for the LMDB embedded database library, offering both an error-checked procedural interface and an object-oriented resource interface with RAII semantics.
+
+
+  license: Unlicense
+  license_file: UNLICENSE
+
+extra:
+  recipe-maintainers: 
+    - DerThorsten
```

You can add myself in.
```suggestion
    - DerThorsten
    - jjerphan
```